releases/openalyx_pruning.py

```diff
@@ -0,0 +1,240 @@
+import json
```

Comment:
```python
"""
This script is designed to run on the SDSC instance of IBL, using a Django installation that has access to both the production database and openalyx.

It is part of a shell script sequence, at the point where the IBL internal alyx database is duplicated on the `public` database. In sequence it will:

- prune the datasets / sessions / subjects to the publicly available datasets
- remove confidential information and nominative information
- create symlinks from the internal IBL disk to the public exposed folder on SDSC
The synchronisation of the AWS public bucket with the public folder is done later on the shell script.
"""
```